### PR TITLE
Chore/Expose PersonNameComponents for AppleSignIn

### PR DESCRIPTION
**Pull request by yllfejziu**
Resolves issue #22.

### CODE REVIEW

File: Sources/Apple/AppleAuthenticator+Models.swift [hunk 1]
```diff
@@ -19,9 +19,15 @@ public extension AppleAuthenticator {
     public let userId: String
     public let token: String
     public let authCode: String
-    public let name: String?
```
Comment (EgzonArifi) [inline Sources/Apple/AppleAuthenticator+Models.swift:22]: I would still keep the `name` so we don't break the clients who already are using it, and add `PersonNameComponents` as a new property. 
cc: @borut-t?

Comment (yllfejziu) [reply Sources/Apple/AppleAuthenticator+Models.swift:22]: We can then rename the computed property `displayName` to `name`.
And rename `name` to `nameComponents`.
What do you say?

Comment (EgzonArifi) [reply Sources/Apple/AppleAuthenticator+Models.swift:22]: sounds good

**Review by EgzonArifi — APPROVED** [review]
LGTM 👌

**Review by nagavcindriqim — APPROVED** [review]

**Review by borut-t — APPROVED** [review]
I like it 👌

**Pull request merged**